core-services/importExport/src/main/java/org/silverpeas/core/importexport/versioning/VersioningImport.java

```diff
@@ -234,13 +216,13 @@ private SimpleDocument isDocumentExist(List<SimpleDocument> documents, Attachmen
             boolean isPublic = version.getType() == DocumentVersion.TYPE_PUBLIC_VERSION;
             if (isPublic) {
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png 'Severity: MINOR') Refactor this code to not nest more than 4 if/for/while/switch/try statements. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://integration.silverpeas.org/sonar/coding_rules#rule_key=squid%3AS134)